test/e2e/flask/btc/btc-account-overview.spec.ts

```diff
@@ -30,15 +29,13 @@ describe('BTC Account - Overview', function (this: Suite) {
           css: '[disabled]',
         });
 
-        const buySellButton = await driver.waitForSelector(
-          '[data-testid="coin-overview-buy"]',
-        );
-        assert.equal(await buySellButton.isEnabled(), true);
+        // buy sell button
+        await driver.findClickableElement('[data-testid="coin-overview-buy"]');
 
-        const portfolioButton = await driver.waitForSelector(
+        // receive button
+        await driver.findClickableElement(
           '[data-testid="coin-overview-receive"]',
```

Comment:
extra note: the name of the const is miss-leading, as this is not the portfolio link but the receive button